lib/tasks/bower-install.js

```diff
@@ -3,68 +3,97 @@
 // Runs `bower install` in cwd
 var path       = require('path');
 var existsSync = require('exists-sync');
+var execa = require('execa');
 var Promise    = require('../ext/promise');
 var Task       = require('../models/task');
 var formatPackageList = require('../utilities/format-package-list');
+var resolve = Promise.denodeify(require('resolve'));
```

Comment:
why use sync when we can use the async version without blocking the progress indicator?